weldx/schemas/weldx.bam.de/weldx/core/mathematical_expression-0.1.0.yaml

```diff
@@ -32,7 +32,9 @@ properties:
     description: |
       List of constant parameters that to be set in the mathematical expression.
     type: object
-    wx_property_tag: "asdf://weldx.bam.de/weldx/tags/units/quantity-0.1.*"
+    anyOf:
+      - wx_property_tag: "asdf://weldx.bam.de/weldx/tags/core/data_array-0.1.*"
+      - wx_property_tag: "asdf://weldx.bam.de/weldx/tags/units/quantity-0.1.*"
```

Comment:
I would prefer if we don't have data_array here as it is very verbose, that should still be possible right? @vhirtham

Reply:
Well, the solution would be to not use data arrays as parameters if you don't need to. But if I have a multi-dimensional equation that uses data arrays as parameters to make my life easier, I can't store it in asdf unless I sort out the dimensions myself and restore the parameters as quantities.